src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,62 @@
 package mate.academy;
 
+import java.math.BigDecimal;
+import mate.academy.dao.BookDao;
+import mate.academy.lib.Injector;
+import mate.academy.model.Book;
+import mate.academy.service.BookService;
+import mate.academy.service.BookServiceImpl;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.academy");
+    private static BookService service;
+
     public static void main(String[] args) {
+        BookDao bookDao = (BookDao) injector.getInstance(BookDao.class);
+        service = new BookServiceImpl(bookDao);
+
+        testCreate();
+        testFindAll();
+        testFindById();
+        testUpdate();
+        testDelete();
+        testFindAll();
+
+        service.deleteAll();
+    }
+
+    private static void testCreate() {
+        System.out.println("\nCREATE");
+        Book bookHarry = new Book("Harry", new BigDecimal(1222));
```

Comment:
Let's create List<Book> books = List.of(new Book(.... and use it in books.stream().map(...